website/src/libs/api-v3/amazonPayments.js

```diff
@@ -70,24 +46,15 @@ api.authorize = (inputSet) => {
   });
 };
 
-api.closeOrderReference = (inputSet) => {
-  let amzPayment = connect(amazonPayments);
-  return new Promize((resolve, reject) => {
-    amzPayment.offAmazonPayments.closeOrderReference(inputSet, (err, response) => {
-      if (err) return reject(err);
-      return resolve(response);
-    });
-  });
-};
-
-api.executePayment = (inputSet) => {
```

Comment:
anyway it's used here https://github.com/HabitRPG/habitrpg/blob/develop/website/src/controllers/payments/amazon.js#L123